packages/block-library/src/gallery/use-get-media.js

```diff
@@ -14,42 +15,24 @@ import { store as coreStore } from '@wordpress/core-data';
  * @return {Array} An array of media info options for each gallery image.
  */
 export default function useGetMedia( innerBlockImages ) {
```

Comment:
This is a nice cleanup - thanks for doing this  - I think a lot of the superfluous code in this hook was a hangover from when we were making individual `getMedia` calls for each image before the `getMediaItems` endpoint was updated to allow us to do it in one call - much cleaner and simpler now, and worked as expected in my testing